plugins/woocommerce/tests/e2e-pw/bin/woocommerce-cleanup.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/*
+Plugin Name: WooCommerce Cleanup
+Description: Resets WooCommerce site to testing start state.
+Version: 1.0
+Author: Solaris Team
+*/
+
+if (!defined('ABSPATH')) {
+	exit; // Exit if accessed directly
+}
+
+function wc_cleanup_reset_site() {
+	// Remove all coupons
+	$coupons = get_posts(array('post_type' => 'shop_coupon', 'numberposts' => -1));
+	foreach ($coupons as $coupon) {
+		wp_delete_post($coupon->ID, true);
+	}
+
+	// Remove all orders
+	$orders = get_posts(array('post_type' => 'shop_order', 'numberposts' => -1));
+	foreach ($orders as $order) {
+		wp_delete_post($order->ID, true);
+	}
+
+	// Remove all products
+	$products = get_posts(array('post_type' => 'product', 'numberposts' => -1));
+	foreach ($products as $product) {
+		wp_delete_post($product->ID, true);
+	}
+
+	// Delete all product categories
+	$product_categories = get_terms('product_cat', array('hide_empty' => false));
+	foreach ($product_categories as $product_category) {
+			wp_delete_term($product_category->term_id, 'product_cat');
+	}
+
+	// Delete all product tags
+	$product_tags = get_terms('product_tag', array('hide_empty' => false));
+	foreach ($product_tags as $product_tag) {
+			wp_delete_term($product_tag->term_id, 'product_tag');
+	}
+
+	// Delete all product attributes
+	$product_attributes = wc_get_attribute_taxonomies();
+	foreach ($product_attributes as $attribute) {
+			wp_delete_term($attribute->attribute_id, 'pa_' . $attribute->attribute_name);
+	}
+
+	// Delete all product reviews
+	$comments = get_comments(array('post_type' => 'product'));
+	foreach ($comments as $comment) {
+			wp_delete_comment($comment->comment_ID, true);
+	}
+
+	// Remove all non-WooCommerce pages
+	$pages = get_posts(array('post_type' => 'page', 'numberposts' => -1));
+	$ignore_titles = array('Cart', 'Checkout', 'My account', 'Shop', 'Refund and Returns Policy', 'Privacy Policy', 'Sample Page');
+
+	foreach ($pages as $page) {
+		if (!has_shortcode($page->post_content, 'woocommerce') && !in_array($page->post_title, $ignore_titles)) {
+			wp_delete_post($page->ID, true);
+		}
+
```

Comment:
Should we remove all posts too?

Reply:
I didn't do anything with users either... could probably make sure the admin and customer users are there.

Reply:
Can we ensure all the taxes are removed?

Reply:
Done